.fixtures.yml

```diff
@@ -3,5 +3,7 @@ fixtures:
     stdlib:
       repo: 'https://github.com/puppetlabs/puppetlabs-stdlib.git'
       ref: '4.6.0'
+    concat:
+      repo: 'https://github.com/puppetlabs/puppetlabs-concat.git'
```

Comment:
please specify the version. This is needed to ensure that testing is consistent.